package/com.unity.formats.usd/Runtime/Scripts/IO/Materials/ShaderExporterBase.cs

```diff
@@ -60,6 +63,22 @@ public class ShaderExporterBase {
 #endif
     }
 
+    static TextureReaderSample.WrapMode WrapMode(TextureWrapMode wrap)
```

Comment:
This probably belongs in TextureReaderSample, since the conversion is likely to be needed again.

I can move after merge, if you prefer, just let me know.

Reply:
Done